tests/search/struct_and_map_types/struct_summary_field_with_explicit_source.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'search_test'
+
+class StructSummaryFieldWithExplicitSource < SearchTest
+  def setup
+    set_owner('toregge')
+    set_description("Test struct summary field with explicit source")
+    @testdir = selfdir + "struct_summary_field_with_explicit_source"
+  end
+
+  def get_app
+    sc = SearchCluster.new('test')
```

Comment:
I don't think an explicit SearchCluster is needed. Please simplify to:
`SearchApp.new.sd("#{@testdir}/test.sd")`